test/e2e/internal/notation/file.go

```diff
@@ -3,33 +3,16 @@ package notation
 import (
 	"encoding/json"
 	"io"
-	"io/fs"
 	"os"
-	"path/filepath"
 )
 
-// copyDir copies the source directory to the destination directory
-func copyDir(src, dst string) error {
-	return filepath.WalkDir(src, func(srcPath string, d fs.DirEntry, err error) error {
-		if err != nil {
-			return err
-		}
-		// generate the dst path
-		relPath, err := filepath.Rel(src, srcPath)
-		if err != nil {
-			return err
-		}
-		dstPath := filepath.Join(dst, relPath)
-
-		if d.IsDir() {
-			return os.MkdirAll(dstPath, os.ModePerm)
-		}
-		return copyFile(srcPath, dstPath)
-	})
-}
-
 // copyFile copies the source file to the destination file
 func copyFile(src, dst string) error {
+	si, err := os.Stat(src)
+	if err != nil {
+		return err
+	}
```

Comment:
It is better to call `in.Stat()` then `os.Stat()` as the former uses file descriptor (fd).

Reply:
Updated.